.changeset/dull-webs-jump.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Improves the font files hashes in development when using the experimental Fonts API to be more explicit
```

Comment:
What kind of changes/different behaviour **the user will see**? Based on the tests, I don't see any user-facing change. This seems more a refactor.

However, if the user sees a change, we should document that. From this changeset, that's not clear.